etc/rucio.cfg.template

```diff
@@ -1,6 +1,7 @@
 [common]
 logdir = /var/log/rucio
 loglevel = DEBUG
+logformat = %%(asctime)s\t%%(process)d\t%%(levelname)s\t%%(message)s
```

Comment:
Does the `config_get` actually unquote the `%%`?

Reply:
Yes. With a single % it doesn't work. Another problem is the \t. It's not converted to an actual tab but it ends up as \t in the logs. I still have to figure out how to do that.